08-Log/readme.md

```diff
@@ -92,19 +86,18 @@ A       MyWindowsApp/code.txt
 
 We can also add `-p` flag and let git list diff for complete history of the file across renames.
 
-* ```git log --follow -p MyWindowsApp/dll.txt```
-
 ```
+> git log --follow -p MyWindowsApp/dll.txt
 commit eeab96ec34708b47a7567cf78059b25b4edb45a6
 Author: Wolf <wolf@wolfsden.cz>
 Date:   Wed Mar 30 16:21:43 2016 +0200
 
     moved code into dll
 ```
 
-* ```diff --git a/MyWindowsApp/code.txt b/MyWindowsApp/dll.txt```
-
 ```
+> diff --git a/MyWindowsApp/code.txt b/MyWindowsApp/dll.txt
```

Comment:
I have no idea what and why you did here. Why are you separating output of `git log -follow -p` into multiple parts? imho it's a bit confusing